sample_analyze_receipts.py

```diff
@@ -0,0 +1,184 @@
+       export DOCUMENTINTELLIGENCE_ENDPOINT=<yourEndpoint>
+       • This is a temporary environment variable setting method that only lasts until you close the terminal session. 
+       • To set an environment variable permanently, visit: https://aka.ms/set-environment-variables-for-Linux
+       
+    ------Set up your programming environment------
+    At a command prompt,run the following code to install the Azure AI Document Intelligence client library for Python with pip:
+    pip install azure-ai-documentintelligence --pre
+    
+    ------Create your Python application------
+    1) Create a new Python file called "sample_analyze_layout.py" in an editor or IDE.
+    2) Open the "sample_analyze_layout.py" file and insert the provided code sample into your application.
+    3) At a command prompt, use the following code to run the Python code: 
+       python sample_analyze_receipts.py
+"""
+
+import os
+
+
+def format_price(price_dict):
+    return "".join([f"{p}" for p in price_dict.values()])
+
+
+def analyze_receipts():
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.documentintelligence import DocumentIntelligenceClient
+    from azure.ai.documentintelligence.models import AnalyzeResult, AnalyzeDocumentRequest
+
+    # For how to obtain the endpoint and key, please see PREREQUISITES above.
+    endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+    key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+    document_intelligence_client = DocumentIntelligenceClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+
+    # Analyze a document at a URL:
+    receiptUrl = "https://raw.githubusercontent.com/Azure-Samples/cognitive-services-REST-api-samples/master/curl/form-recognizer/rest-api/receipt.png"
+    # Replace with your actual receiptUrl:
+    # If you use the URL of a public website, to find more URLs, please visit: https://aka.ms/more-URLs 
+    # If you analyze a document in Blob Storage, you need to generate Public SAS URL, please visit: https://aka.ms/create-sas-tokens
+    poller = document_intelligence_client.begin_analyze_document(
+        "prebuilt-receipt",
+        AnalyzeDocumentRequest(url_source=receiptUrl)
+    )       
+
+    # # If analyzing a local document, remove the comment markers (#) at the beginning of these 8 lines.
+    # # Delete or comment out the part of "Analyze a document at a URL" above.
+    # # Replace <path to your sample file>  with your actual file path.
+    # path_to_sample_document = "<path to your sample file>"
+    # with open(path_to_sample_document, "rb") as f:
+    #     poller = document_intelligence_client.begin_analyze_document(
+    #         "prebuilt-receipt", analyze_request=f, locale="en-US", content_type="application/octet-stream"
+    #     )
+    receipts: AnalyzeResult = poller.result()
+
+    # [START analyze_receipts]
+    # To find more schemas, please refer to: https://aka.ms/azsdk/documentintelligence/receiptfieldschema
```

Comment:
this link seems not work any more, should be something like this: https://learn.microsoft.com/en-us/azure/ai-services/document-intelligence/concept-receipt?view=doc-intel-4.0.0#field-extraction

Reply:
Replaced with field-related links